src/pages/product.html

```diff
@@ -0,0 +1,280 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="../../assets/css/output.css">
+  <link rel="stylesheet" href="../../assets/fontawesome/css/all.min.css">
+  <link href="https://fonts.cdnfonts.com/css/lato" rel="stylesheet">
+  <link href="https://fonts.cdnfonts.com/css/open-sans" rel="stylesheet">
+</head>
+
+<body>
+    <div id="header"></div>
+
+    <div class="container mx-auto max-w-7xl text-sm px-2.5">
+        <div class="px-2.5">
+            <div class="py-5 text-sm border-b border-gray-200">
+                <nav id="breadcrumb" class="flex list-none gap-2">
+        
+                </nav>
+            </div>
+        </div>
+    </div>
+
+    <img src="../../assets/images/product-banner.jpg" alt="Banner" class="w-full sm:block hidden">
+
+    <div class="container mx-auto max-w-7xl mb-24 text-sm px-2.5 flex flex-col md:flex-row mt-10">
+        <div class="w-full md:w-1/4">
+            <div class="mb-5">
+                <div class="text-xl font-bold mb-2 border-b pb-2" data-i18n="product.catalog"></div>
+                <ul class="space-y-2">
+                    <li class="group active">
+                        <div class="flex justify-between cursor-pointer border-b pb-2 group-hover:text-primary-color text-primary-color">
+                            <span data-i18n="product.cosmetics"></span>
+                            <i class="fa fa-angle-right group-hover:rotate-90 rotate-90"></i>
+                        </div>
+                        <ul class="max-h-0 overflow-hidden transition-all duration-300 group-hover:max-h-40 max-h-40 list-disc pl-7">
```

Comment:
```suggestion
                        <ul class="max-h-0 overflow-hidden transition-all duration-300 group-hover:max-h-none max-h-40 list-disc pl-7">
```
Chỗ này e dùng thử group-hover:max-h-none để khi hover thì sẽ tự động lấy chiều cao của content thử nha